File: 0001-two-sum/0001-two-sum.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        #using hashMap
        obj = {}
        
        for i in range(len(nums)):
            if nums[i] in obj:
                return [obj[nums[i]], i]
            
            obj[target - nums[i]] = i

File: 0001-two-sum/0001-two-sum.go
func twoSum(nums []int, target int) []int {
    hash := make(map[int]int)
    
    for i, v := range nums {
        if val, ok := hash[v]; ok {
            return []int{val, i}
        }
        hash[target - v] = i
    }
    
    
    return []int{}
}